source/agora/node/Node.d

```diff
@@ -223,10 +223,13 @@ public class Node : API
         return this.config.node.key_pair.address;
     }
 
-    /// GET: /network_info
-    public override NetworkInfo getNetworkInfo () pure nothrow @safe @nogc
+    /// GET: /node_info
+    public override NodeInfo getNodeInfo () pure nothrow @safe @nogc
     {
-        return this.network.getNetworkInfo();
+        // NetworkInfo only brings network-related information.
+        NodeInfo node_info = this.network.getNetworkInfo();
+        node_info.is_validator = this.config.node.is_validator;
```

Comment:
I wanted to separate from `network.NetworkInfo`
Is this where the initialization takes place?